tutorials/migration.md

```diff
@@ -4,96 +4,846 @@ layout: intro
+the bindings are included in the initial installation (included does not mean
+installed) or whether openHAB 2 needs to connect to and download the latest
+bindings on demand. Offline is a good choice if your openHAB server does not
+have access to the internet all the time.
+
+If you plan on running openHAB 1.x as your primary while migrating make sure to
+disable openHAB 2 from starting automatically.
+
+`sudo systemctl disable openhab2.service`
+
+##Prepare openHAB 2 for the Existing openHAB 1.x Configuration
+
+Start your newly installed openHAB 2 instance per the instruction in the 
+installation guide for your platform and navigate to 
+`http://<openhab server ip>:8080` where `<openhab server ip>` is the address of
+the server where openHAB 2 is running.
+
+Congratulations, you have a running openHAB 2! See 
+[Concepts](../concepts/index.html) and below for important differences between
+openHAB 1.x and openHAB 2.
+
+Theoretically one should be able to just enable the 
+[openHAB 1.x Compatibility Layer](../developers/development/compatibilitylayer.html#how-to-use-openhab-1x-add-ons-that-are-not-part-of-the-distribution),
+copy over the existing add-ons and config files and have it work. However, while 
+the compatibility layer is very good and very capable, this approach will 
+result in errors and end up being more work than the steps below. Therefore the
+first steps will be to get it running using the 1.9 version of the bindings 
+installed through openHAB 2's new add-on management system.
+
+There are three approaches one can use to install and configure bindings, an 
+all text based one, using the Karaf console, or using one of the administration 
+GUIs (i.e. PaperUI or Habmin 2). While all three approaches are presented below, 
+the text based approach is the recommended one for those coming from openHAB 1.x 
+as it will be more familiar and flexible.
+
+###Text Based
+
+In your configurations folder for openHAB 2 (`/etc/openhab2/conf` on apt-get 
+installed instances) there is a new folder named `services`. In this folder you
+will find a file named `runtime.cfg`. This file contains global openHAB 2 
+parameters. Open this file for editing and you will find a number of parameters 
+you may want to set. In particular for this tutorial we recommend setting:
+
+- `org.eclipse.smarthome.persistence:default=mapdb` - default persistence, use
+whatever was set in `persistence:default` in openhab.cfg
+- `autoapprove:enabled=false` - makes it so you must manually approve 
+automatically discovered Things in the Inbox (see below)
+- `org.eclipse.smarthome.links:autoLinks=false` - prevents openHAB from 
+automatically creating an Item for each automatically discovered Thing.
+
+Next open for editing `addons.cfg`. This file contains parameters related to 
+add-ons including allowing one to choose which add-ons to install. For now just
+set the following parameters:
+
+- `package = minimal` - minimal or standard are good choices here. Standard
```

Comment:
The expert package sounds like the sort of package that migration users would be looking for and I think this tutorial won't need to change too much to use this.